File: Bachelor/Semester4/Systems_for_Design_and_Implementation/C#/ProiectCShMPP/Domain/domain/dtos/TripDataDTO.cs
using System;

namespace ProiectCShMPP.domain.dtos
{
    [Serializable]
    public class TripDataDTO
    {
        public string Destination
        {
            get;
            set;
        }

        public DateTime From
        {
            get;
            set;
        }

        public DateTime To
        {
            get;
            set;
        }

        public override string ToString()
        {
            return @"TripDataDTO" + Destination + " " + From + " " + To;
        }
    }
}

File: Bachelor/Semester4/Systems_for_Design_and_Implementation/Java/Domain/src/main/java/domain/dtos/TripDataDTO.java
package domain.dtos;

import java.io.Serializable;
import java.time.LocalDateTime;

public class TripDataDTO implements Serializable {
    private final String destination;
    private final LocalDateTime from;
    private final LocalDateTime to;

    public TripDataDTO(String destination, LocalDateTime from, LocalDateTime to) {
        this.destination = destination;
        this.from = from;
        this.to = to;
    }

    public String getDestination() {
        return destination;
    }

    public LocalDateTime getFrom() {
        return from;
    }

    public LocalDateTime getTo() {
        return to;
    }
}
